agent/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ elseif (TARGET_PLATFORM STREQUAL "linux")
     set(BEEROCKS_TMP_PATH "/tmp/$ENV{USER}/beerocks")
     set(BEEROCKS_NOTIFY 0)
     set(BEEROCKS_BRIDGE_IFACE "br-lan")
-    set(BEEROCKS_BH_WIRE_IFACE "eth0")
+    set(BEEROCKS_BH_WIRE_IFACE "prplmesh-eth0")
```

Comment:
NIC names are (in some cases) limited to 11 characters, so this will be truncated to `prplmesh-et`. So `prplmesh` must be abbreviated. @wcloetens suggested `sim0` on Slack, but I'd make it `sim-eth0`.